lib/elastic_whenever/task/target.rb

```diff
@@ -37,25 +42,58 @@ def initialize(option, cluster:, definition:, container:, commands:, rule:, role
         @commands = commands
         @rule = rule
         @role = role
+        @assign_public_ip = option.assign_public_ip
+        @launch_type = option.launch_type
+        @platform_version = option.platform_version
+        @security_groups = option.security_groups
+        @subnets = option.subnets
         @client = Aws::CloudWatchEvents::Client.new(option.aws_config)
       end
 
       def create
-        client.put_targets(
-          rule: rule.name,
-          targets: [
-            {
-              id: Digest::SHA1.hexdigest(commands.join("-")),
-              arn: cluster.arn,
-              input: input_json(container, commands),
-              role_arn: role.arn,
-              ecs_parameters: {
-                task_definition_arn: definition.arn,
-                task_count: 1,
+        if launch_type == 'FARGATE'
+          client.put_targets(
+            rule: rule.name,
+            targets: [
+              {
+                id: Digest::SHA1.hexdigest(commands.join("-")),
```

Comment:
@avinson @wata727 	I've been testing your pull request on our Fargate cluster. The `id` being a hash is the most troubling thing about this. Trying to identify the logs by the rake task is a matter of going through each of the events until you find it. 

Why not strip out all invalid characters and just use the command as the id?